specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/2017-06-15-preview/hanaonazure.json

```diff
@@ -0,0 +1,380 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "HanaManagementClient",
+        "description": "The SAP HANA on Azure Management Client.",
+        "version": "2017-06-15-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "securityDefinitions": {
+        "azure_auth": {
+            "type": "oauth2",
+            "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+            "flow": "implicit",
+            "description": "Azure Active Directory OAuth2 Flow",
+            "scopes": {
+                "user_impersonation": "impersonate your user account"
+            }
+        }
+    },
+    "paths": {
+        "/providers/Microsoft.HanaOnAzure/operations": {
+            "get": {
+                "tags": [
+                    "HanaOnAzure"
+                ],
+                "operationId": "HanaOperations_List",
```

Comment:
our naming convention here is `Operations_List`, since the client that this operation is on will already contain the "subject", i.e. "Hana" in this case

Reply:
Updated to just Operations_List